scanners/typo3scan/templates/typ3scan-parse-definition.yaml

```
@@ -0,0 +1,9 @@
+apiVersion: execution.securecodebox.io/v1
+kind: ParseDefinition
+metadata:
+  name: typo3scan-json
+spec:
+  image: "{{ .Values.parser.image.repository }}:{{ .Values.parser.image.tag | default .Chart.Version }}"
+  imagePullSecrets:
+  - name: dockerhub-token
+  ttlSecondsAfterFinished: 60
```

The parse definition does not match with the standards from other scanners, e.g. gitleaks or nmap. Is there a reason for that?
```suggestion
spec:
  image: "{{ .Values.parser.image.repository }}:{{ .Values.parser.image.tag | default .Chart.Version }}"
  ttlSecondsAfterFinished: {{ .Values.parser.ttlSecondsAfterFinished }}
  env: 
    {{- toYaml .Values.parser.env | nindent 4 }}
```
(copied from nmap)